src/components/MainDashboard/OpenPerscription/OpenPerscription.js

```diff
@@ -0,0 +1,86 @@
+import style from "./OpenPerscription.module.css";
+import Accordion from 'react-bootstrap/Accordion';
+
+const OpenPerscription = () => {
+    return (
+        <section className={style["section-openPerscription"]}>
```

Comment:
Generally, it's not a good practice to mix `kebab-case` and `camelCase`, or any other writing styles. We should stick with `kebab` only for class names. 
There's also a typo in the word _prescription_ - 'e' and 'r' have their places switched. 
In conclusion, the class nams should be rewritten as "`section-open-prescription`"

Reply:
Done